tests/phpunit/integration/Core/Admin/PointerTest.php

```diff
@@ -134,6 +134,21 @@ public function test_is_active( array $args, $expected ) {
 		}
 	}
 
+	public function test_get_buttons() {
+		$buttons = 'function(event, container) { return jQuery("<button>OK</button>"); }';
```

Comment:
The fact that this test passes with this input somewhat sets the expectation that this method is intended to return JS and would work although that's no longer the case.